frontend/src/bundles/common/components/input/input.tsx

```diff
@@ -23,17 +26,29 @@ const Input = <T extends FieldValues>({
     name,
     placeholder = '',
     type = 'text',
+    disabled = false,
 }: Properties<T>): JSX.Element => {
     const { field } = useFormController({ name, control });
-
     const error = errors[name]?.message;
     const hasError = Boolean(error);
-
     return (
-        <label>
-            <span>{label}</span>
-            <input {...field} type={type} placeholder={placeholder} />
-            {hasError && <span>{error as string}</span>}
+        <label className={styles.label}>
+            <span className={styles.labelName}>{label}</span>
+            <input
```

Comment:
I'd like to have BaseInput component, and FormGroup component with validation, label, etc...